app/admin/widgets/edit.php

```diff
@@ -70,7 +70,7 @@
 	<!-- params -->
 	<tr>
 	    <td><?php print _('Parameters'); ?></td>
-	    <td><input class="form-control input-sm input-w-250" type="text" name="wparams" value="<?php print $Admin->strip_xss(@$w['wparams']); ?>" <?php if($_POST['action'] == "delete") print "readonly"; ?>></td>
+	    <td><input class="form-control input-sm input-w-250" type="text" name="wparams" value="<?php print str_replace('&amp;','&',$Admin->strip_xss(@$w['wparams'])); ?>" <?php if($_POST['action'] == "delete") print "readonly"; ?>></td>
```

Comment:
str_replace isn't right here.
The issue is strip_xss calls htmlspecialchars() on data that's already been through htmlentities() via the PDO layer.
It's being double escaped.

Reply:
I understand it's being double-escaped, but undoing 1 of the escapes is enough for the browser to undo the other one and present it correctly.

Reply:
Also, i feel `str_replace` is the most surgical way to do it without un-escaping other undesirable things.